templates/nginx.conf

```diff
@@ -90,6 +98,13 @@ server {
       proxy_set_header X-Real-IP $remote_addr;
       proxy_set_header Host $host;
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+
+      # Response cache for unauthenticated users
+      proxy_cache lemmy_cache_{{domain}};
+      proxy_cache_use_stale error timeout http_500 http_502 http_503 http_504;
```

Comment:
adding `updating` can also help here to reduce requests to upstream, when an object expires out of the cache.